Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -459,6 +459,15 @@ public class PBXProjGenerator {
             }
             return !linkingAttributes.isEmpty ? ["ATTRIBUTES": linkingAttributes] : nil
         }
+        
+        func isStaticDependency(for carthageDependency: Dependency) -> Bool {
```

Comment:
Could this instead be an extension on `Dependency`?

Reply:
This is a local function inside `generateTarget` function. So it is a narrower scope than `fileprivate extension`. This is why I use function instead of extension.

Reply:
I agree about moving this to an extension. If we move static/linkType to Dependency itself like in the other comment we don't even need this function. 

Otherwise if this crashes when used with other types it should be called `carthageDependencyIsStatic`, otherwise it should probably just return false for other cases